packages/srt/srt.0.3.0/opam

```diff
@@ -0,0 +1,47 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Binding for the Secure, Reliable, Transport protocol library"
+description: """
+Secure Reliable Transport (SRT) is an open source transport technology
+that optimizes streaming performance across unpredictable networks, such
+as the Internet.
+This package provides OCaml bindings to the C implementation library.
+"""
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "GPL-2.0-only"
```

Comment:
Note that according to your files (e.g. https://github.com/savonet/ocaml-srt/blob/a10d0785c188236c3c11e70d53f4ce9a021f09e1/src/srt.mli#L9) this should be `GPL-2.0-or-later` instead